### DISCUSSION TITLE
About `base` config

### DISCUSSION THREAD

**Discussion by bp404** [web-infra-dev/rspress#1197] [Q&A] [answered]
编译的文档有本地打开的需求。因本地存放位置未知，如打开后浏览器url为file:///Users/test/Desktop/docs/index.html或file:///docs/index.html等，针对路径不确定这种情况base(项目基础路径配置)参数是否支持？即把index.html的所在目录作为base目录

**Comment by Timeless0911** [comment, marked as answer]
This is definitely not an issue that can be solved by the base directory configuration. Accessing web pages using the file protocol is definitely not a right choice. Opening locally requires starting a server like what `rspress preview` command do.
